.eslintrc.js

```diff
@@ -35,7 +35,15 @@ module.exports = {
     "func-names" : ["warn", "as-needed"],
     "no-underscore-dangle": ["error", {
       "allowAfterThis": true
-    }]
+    }],
+    "quotes" : ["error", "single", { "allowTemplateLiterals": true }],
+    "arrow-parens" : "off",
+    "arrow-body-style" : "off",
+    "func-names" : "off",
```

Comment:
@jniles
I agree, but this rule lead me to even a function which process a promise 
```
.then(function (){
});
```
produces an error

```
.then(function someName(){
});
```
passes but produces an error in the execution

So do you think, I should declare those function on the top of their scopes?